### PR TITLE
Remove some code

### PR DESCRIPTION
**Pull request by mattcaswell**
This commit removes the contribution of a user that we cannot trace to
gain their consent for the licence change.

After this commit the various IS_\*() macros in the auto-generated file
conf_def.h may incorrectly return true if the supplied character has its
most significant bit set. The IS_\*() macros should be able to correctly
handle 8-bit characters. Note that UTF-8 support is not a requirement.

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Comment by richsalz** [comment]
-1 pending OMC discussion.

File: crypto/conf/keysets.pl [hunk 1]
```diff
@@ -107,22 +103,21 @@
...
 
 my $i;
 
-print "static const unsigned short CONF_type_default[256] = {";
-for ($i = 0; $i < 256; $i++) {
+print "static const unsigned short CONF_type_default[128] = {";
+for ($i = 0; $i < 128; $i++) {
     print "\n   " if ($i % 8) == 0;
-    printf " 0x%04X,", $V_def[$i];
+    printf " 0x%03X,", $V_def[$i];
```
Comment (richsalz) [inline crypto/conf/keysets.pl]: This is purely a cosmetic change and I don't see the need for it.  It just makes the diff needlessly bigger.

File: crypto/conf/keysets.pl [hunk 2]
```diff
@@ -107,22 +103,21 @@
...
 }
 print "\n};\n\n";
 
-print "static const unsigned short CONF_type_win32[256] = {";
-for ($i = 0; $i < 256; $i++) {
+print "static const unsigned short CONF_type_win32[128] = {";
+for ($i = 0; $i < 128; $i++) {
     print "\n   " if ($i % 8) == 0;
-    printf " 0x%04X,", $V_w32[$i];
+    printf " 0x%03X,", $V_w32[$i];
```
Comment (richsalz) [inline crypto/conf/keysets.pl]: Same comment; leave it as 04.

**Comment by paulidale** [comment]
When putting equivalent code back, would it make sense to move this into `crypto/ctype.c`?

**Comment by mattcaswell** [comment]
> When putting equivalent code back, would it make sense to move this into crypto/ctype.c?

I'm not going to comment on the right solution. I'm hoping others not familiar with the code will come up with something.

File: crypto/conf/keysets.pl (hunk 1 above)
Comment (mattcaswell) [reply crypto/conf/keysets.pl]: Ok, on the basis that this is trivial and purely cosmetic I have left this.

File: crypto/conf/keysets.pl (hunk 2 above)
Comment (mattcaswell) [reply crypto/conf/keysets.pl]: As above.

**Comment by mattcaswell** [comment]
I have rebased this to pick up the re-enabled tests in master, and made the changes as requested above.

I understand from @richsalz that his -1 on this no longer applies, so please review @openssl.

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**